charts/nri-metadata-injection/templates/deployment.yaml

```diff
@@ -56,7 +57,7 @@ spec:
       volumes:
       - name: tls-key-cert-pair
         secret:
-          secretName: {{ template "nri-metadata-injection.fullname" . }}
+          secretName: {{ template "nri-metadata-injection.fullname" . }}-admission
```

Comment:
This change breaks the cert-manager setup, which is creating the old name

https://github.com/newrelic/helm-charts/blob/b4d3421a21d2291f664547f1f190495fbd40c2ef/charts/nri-metadata-injection/templates/cert-manager.yaml#L41

Reply:
I fixed the name to be consistent. Hopefully that won't break cert-manager.